metadata-checker/popup/metadata.js

```diff
@@ -0,0 +1,138 @@
+function onMetaData(metaData) {
+  metaData = metaData[0];
```

Comment:
It's probably worth commenting this line, e.g.

```
Because tabs.executeScript() may inject the script into more than one frame,
it is resolved with an array of objects, one for each frame in which it ran,
representing the result of executing the script in that frame.
```
...or something like that.